### PR TITLE
DPC-1240 add swagger-ui

### PR DESCRIPTION
**Pull request by MrBilnon**
### Proposed Changes
* create a openapi 3.0 spec file
* bring in swagger-ui dist from swagger-ui github
*  edit index.html to use the openapi spec file
* add route to chi router to serve the static files

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### Acceptance Validation

Screen Shot 2021-04-27 at 12 10 03 AM

### CODE REVIEW

File: dpc-go/dpc-api/swaggerui/dpc-api-swagger.json [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+  "paths": {
+    "/v2/metadata": {
+      "get": {
+        "tags": [
+          "Metadata"
+        ],
+        "summary": "Get FHIR Server Capability Statement",
+        "description": "Returns the FHIR Capabilities statement for DPC",
+        "operationId": "metadata",
+        "responses": {
```
Comment (jonfulk) [inline dpc-go/dpc-api/swaggerui/dpc-api-swagger.json:38]: Do we only return 200s right now for this and the Organization requests? If not, I think we need the other possible status codes in the `responses` sections

File: dpc-go/dpc-api/src/router/router.go [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
 // NewDPCAPIRouter function that builds the router using chi
 func NewDPCAPIRouter(oc v2.Controller, mc v2.ReadController, gc v2.CreateController) http.Handler {
+	fs := http.FileServer(http.Dir("../swaggerui/"))
+	sh := http.StripPrefix("/swaggerui/", fs)
+
 	r := chi.NewRouter()
 	r.Use(middleware2.Logging())
-	r.Use(middleware.SetHeader("Content-Type", "application/fhir+json; charset=UTF-8"))
 	r.Use(middleware2.Sanitize)
+	r.Use(middleware.SetHeader("Content-Type", "application/fhir+json; charset=UTF-8"))
```
Comment (dhgreene) [inline dpc-go/dpc-api/src/router/router.go:21]: Does this set our Swagger `Content-Type` to `application/fhir+json`?  Is that what we want for the Swagger page?

File: dpc-go/dpc-api/swaggerui/dpc-api-swagger.json [hunk 3]
```diff
@@ -0,0 +1,218 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "description": "As patients move throughout the healthcare system, providers often struggle to gain and maintain a complete picture of their medical history.\n        Data at the Point of Care fills in the gaps with claims data to inform providers with secure, structured patient history, past procedures, medication adherence, and more.",
```
Comment (dhgreene) [inline dpc-go/dpc-api/swaggerui/dpc-api-swagger.json:4]: Can we safely break this line?

File: dpc-go/dpc-api/swaggerui/dpc-api-swagger.json [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+    {
+      "name": "Group"
+    },
+    {
+      "name": "Organization"
+    }
+  ],
+  "servers": [
+    {
+      "url": "https://prod.dpc.cms.gov",
```
Comment (dhgreene) [inline dpc-go/dpc-api/swaggerui/dpc-api-swagger.json:25]: Shall we add sandbox here too?

File: dpc-go/dpc-api/src/router/router.go (hunk 2 above)
Comment (MrBilnon) [reply dpc-go/dpc-api/src/router/router.go:21]: hitting it in the browser brings back the html so maybe go or chi is smart enough to know?

File: dpc-go/dpc-api/swaggerui/dpc-api-swagger.json (hunk 3 above)
Comment (MrBilnon) [reply dpc-go/dpc-api/swaggerui/dpc-api-swagger.json:4]: I've tried to break it up in goland and it won't let me.. Not sure what's going on

**Review by jonfulk — DISMISSED** [review]

File: dpc-go/dpc-api/swaggerui/dpc-api-swagger.json (hunk 4 above)
Comment (dhgreene) [reply dpc-go/dpc-api/swaggerui/dpc-api-swagger.json:25]: In fact, given our audience, maybe we should replace this with
```suggestion
      "url": "https://sandbox.dpc.cms.gov",
```

**Comment by MrMorie** [comment]
So much swag in this PR.

LGTM

**Review by dhgreene — APPROVED** [review]
🍎

**Pull request merged**